example_training_data/geotags-labels.csv

```diff
@@ -0,0 +1,10 @@
+﻿0,0
```

Comment:
to lower the repo's memory, I'm currently working on #503 

in the meantime, just keep the labels and samples local, then we will move them to some cloud :)